### ISSUE TITLE
[MPOM-235] Google Custom Search broken

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-parent#431]
**[Piotr Zygielo](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=pzygielo)** opened **[MPOM-235](https://issues.apache.org/jira/browse/MPOM-235?redirect=false)** and commented

Sites

```
Rendered using Apache Maven Fluido Skin 1.7
```

for example:
* http://maven.apache.org/plugins/maven-clean-plugin/clean-mojo.html
* http://maven.apache.org/plugin-tools/maven-plugin-plugin/examples/generate-descriptor.html
* http://maven.apache.org/plugin-tools/maven-plugin-tools-annotations/index.html#Supported_Annotations
  all have Custom Google Search broken in following way:
  After entering search term (here: requirements), no results are presented as in attached apache-search.png.

---

**Affects:** MAVEN-33

**Attachments:**
- [apache-search.png](https://issues.apache.org/jira/secure/attachment/12990642/apache-search.png) (_40.48 kB_)

**Issue Links:**
- [MCLEAN-90](https://issues.apache.org/jira/browse/MCLEAN-90) Custom search broken on pages rendered using Fluido Skin 1.7
 (_**"causes"**_)
- [MPLUGIN-357](https://issues.apache.org/jira/browse/MPLUGIN-357) Custom search broken on pages rendered using Fluido Skin 1.7
 (_**"causes"**_)
- [MRELEASE-1039](https://issues.apache.org/jira/browse/MRELEASE-1039) Custom search broken on pages rendered using Fluido Skin 1.7
 (_**"causes"**_)

**Comment by jira-importer** [comment]
**[Piotr Zygielo](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=pzygielo)** commented

So I found, that http://maven.apache.org/developers/mojo-api-specification.html, rendered with 1.8, works fine.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

So we basically need to update affected sites with newest Fluido Skin.

**Comment by jira-importer** [comment]
**[Piotr Zygielo](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=pzygielo)** commented

[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o) - thanks for checking my report.
I'm not 100% sure it's Fluido or rather site plugin or doxia configuration somewhere (perhaps in one of many parents of given project).
I _"investigated"_  only briefly up to the point where `$sitesearchValue` was observable in code. It seemed two downstream projects I happened to read about at the moment, were affected in the same way.
So it's also possible Fluido is fine but not fully configured.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

It works in the IT `mskins-14_sitesearch`. I will close this as soon as the plugins switch to a new skin version.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This [snippet](https://github.com/apache/maven-parent/blob/master/src/site-docs/site.xml#L53-L55) triggers the value to be empty. It needs to be replaced by:

```
<googleSearch />
```

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [eb2fb5096fc506a15431e67214f8de3d8e351810](https://gitbox.apache.org/repos/asf?p=maven-parent.git;a=commit;h=eb2fb5096fc506a15431e67214f8de3d8e351810).

**Comment by jira-importer** [comment]
**[Sylwester Lachiewicz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slachiewicz)** commented

Also here:
* https://maven.apache.org/pom/maven/
* maven-shade-plugin
